ArmoniK.SDK.Client.Test/src/ArmoniK.SDK.Client.Test.cpp

```diff
@@ -289,3 +293,107 @@ TEST(testSDK, testAddFloat) {
 
   std::cout << "Done" << std::endl;
 }
+
+TEST(testSDK, testStressTest) {
+  std::cout << "Doing some computation!" << std::endl;
+  // Load configuration from file and environment
+  ArmoniK::Sdk::Common::Configuration config;
+  config.add_json_configuration("appsettings.json").add_env_configuration();
+
+  std::cout << "Endpoint : " << config.get("Grpc__EndPoint") << std::endl;
+  if (config.get("Worker__Type").empty()) {
+    config.set("Worker__Type", "End2EndTest");
+  }
+
+  // Create the task options
+  ArmoniK::Sdk::Common::TaskOptions session_task_options("libArmoniK.SDK.Worker.Test.so",
+                                                         config.get("WorkerLib__Version"), "End2EndTest", "StressTest");
+  session_task_options.max_retries = 1;
+
+  // Create the properties
+  ArmoniK::Sdk::Common::Properties properties{config, session_task_options};
+
+  // Create the logger
+  armonik::api::common::logger::Logger logger{armonik::api::common::logger::writer_console(),
+                                              armonik::api::common::logger::formatter_plain(true)};
+
+  // Create the session service
+  ArmoniK::Sdk::Client::SessionService service(properties, logger);
+
+  // Get the created session id
+  std::cout << "Session : " << service.getSession() << std::endl;
+
+  // Create the handler
+  auto handler = std::make_shared<StressTestServiceHandler>(logger);
+
+  const std::uint32_t nbTasks = 5, nbInputBytes = 64000, nbOutputBytes = 8, workloadTimeInMs = 1;
```

Comment:
```suggestion
  const std::uint32_t nbTasks = 100, nbInputBytes = 64000, nbOutputBytes = 8, workloadTimeInMs = 1;
```
I tested in my environnement with a 100 it is not a problem !
Maybe you can change the number of bytes for input and output to the number of elements in those vectors. And then use sizeof to deduce the number of bytes.